interface/super/edit_list.php

```diff
@@ -1238,11 +1311,25 @@ function mysubmit() {
              *   If the edit_options setting of the main list entry is set to 1,
              *    then the list items with edit_options set to 1 will show.
              */
+
+            /**
+             * In case when we are have a get parametr "list_id_container", we are set manually variable list_id
+             * and range from...to if these is set
+             * adding 1 to from and to limits
+             */
+//            $list_from = ($list_from <= 0 ? 0 : $list_from);
+            if ( $list_from > 0 ){
+                $list_from--;
+            }
+            if ( $list_to > 0 ) {
+                $sql_limits = "LIMIT " . escape_limit($list_from) . (intval($list_to) > 0 ? ", " . escape_limit($list_to - $list_from) : "");
+            }
+
             $res = sqlStatement("SELECT lo.*
                          FROM list_options as lo
                          right join list_options as lo2 on lo2.option_id = lo.list_id AND lo2.edit_options = 1
                          WHERE lo.list_id = ? AND lo.edit_options = 1
-                         ORDER BY seq,title", array($list_id));
+                         ORDER BY seq,title ".$sql_limits, array($list_id));
```

Comment:
`$sql_limits` needs to be part of the array that contains `$list_id`

`ORDER BY seq,title ?", array($list_id, $sql_limits));`

Reply:
In the next commit, I will make a correction.

Reply:
Hi @robertdown , I tried to do the example, but the function "qstr", which is called when building a query, adds a quote to each text line.

Reply:
hi @yurige-matrix , Sorry for the delays on my reviews(plan to review both of your PR's very soon). your $sql_limits call in this query is fine. Limits can not be binded, and you already escaped them via the escape_limit() functions above. As an aside, the escape_limit() function ensures that it is only a number.

Reply:
Huh, limits can’t be binded? I learn something new everyday!